ApplicationCode/ProjectDataModel/RimGeoMechCase.cpp

```diff
@@ -80,6 +80,18 @@ void caf::AppEnum<RimGeoMechCase::BiotCoefficientType>::setUp()
     setDefault( RimGeoMechCase::BiotCoefficientType::BIOT_NONE );
 }
 
+template <>
+void caf::AppEnum<RimGeoMechCase::InitialPermeabilityType>::setUp()
+{
+    addItem( RimGeoMechCase::InitialPermeabilityType::INITIAL_PERMEABILITY_FIXED,
+             "INITIAL_PERMEABILITY_FIXED",
+             "Fixed initial permeability" );
```

Comment:
Use capital letter for first letter of important words
Fixed Initial Permeability

Reply:
Fixed (and some others in the GeoMechCase UI).